docs/pages/model_hub/asr.md

```diff
@@ -70,6 +70,55 @@ Here are some other possible configurations for the Whisper deployment:
     )
     ```
 
+### Examples of Transcription from Video
+
+Let's see different transcribe methods in the transcription endpoint class. 
+
+!!! example "Transcribe methods in Aana SDK"
```

Comment:
I would suggest to structure it a list of available methods. No need to mention endpoint here, it's kinda irrelevant since you can run the same code without endpoint as well.